### DISCUSSION TITLE
Virus scanner (Bitdefender) blocks “iib_api_server.exe”

### DISCUSSION THREAD

**Discussion by LordKa-Berlin** [zanllp/infinite-image-browsing#736] [General]
first of all, I think this tool is really great, it's exactly what I've been looking for!

Now to my problem,
when I start the local program of Infinite Image Browsing the virus scanner starts EVERY time, I have already added the program directory to the exceptions. The problem is that the
iib_api_server is copied to a temporary directory in the path 
“c:\users\username\appdata\local\temp\onefile_37928_133735551012044697\iib_api_server.exe”
is copied. Since each time a new folder name “onefile_37928_133735551012044697” is created, the number changes each time, I cannot add this directory to the virus scanner exceptions.
If I add the parent folder temp to the exceptions of the virus scanner, then everything works fine. But I cannot permanently add the folder “c:\users\username\appdata\local\temp\” to the exceptions, that would be data suicide!
So either the server is not created under Temp or the folder name under Temp is given a static folder name so that I can then integrate this path into the acceptance rules of the virus scanner.

**Comment by zanllp** [comment]
The EXE version currently cannot completely resolve this false positive virus issue. You can try using the IIB version of PyInstaller, which has significantly reduced false positives. Alternatively, you can run IIB directly with Python.
